pkgs/top-level/python-packages.nix

```diff
@@ -2272,6 +2272,23 @@ let
     };
   };
 
+  pkginfo = buildPythonPackage rec {
+    version = "1.2.1";
+    name = "pkginfo-${version}";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/p/pkginfo/${name}.tar.gz";
+      sha256 = "0g0g6avplfqw1adzqybbrh1a2z0kfjl8qn3annkrc7w3ibz6sgxd";
+    };
+
+    doCheck = false; # I don't know why, but with doCheck = true it fails.
+
+    meta = {
+      homepage = https://pypi.python.org/pypi/pkginfo;
```

Comment:
The documentation of this package says that platforms is `[Unix, Windows]`, so I assume yes, `all` is correct.

description and longDescription added.